onnxruntime/test/onnx/TestCase.cc

```diff
@@ -403,21 +413,31 @@ static bool read_config_file(const std::basic_string<PATH_CHAR_TYPE>& path, std:
 
 //load tensors from disk
 template <typename PATH_STRING_TYPE>
-static void LoadTensors(const std::vector<PATH_STRING_TYPE>& pb_files,
-                        std::vector<ONNX_NAMESPACE::TensorProto>* input_pbs) {
-  for (size_t i = 0; i != pb_files.size(); ++i) {
-    int tensor_fd;
-    auto st = Env::Default().FileOpenRd(pb_files.at(i), tensor_fd);
-    if (!st.IsOK()) {
-      ORT_THROW("open file '", ToMBString(pb_files.at(i)), "' failed:", st.ErrorMessage());
-    }
-    google::protobuf::io::FileInputStream f(tensor_fd, protobuf_block_size_in_bytes);
-    f.SetCloseOnDelete(true);
-    ONNX_NAMESPACE::TensorProto tensor;
-    if (!tensor.ParseFromZeroCopyStream(&f)) {
-      ORT_THROW("parse file '", ToMBString(pb_files.at(i)), "' failed");
-    }
-    input_pbs->emplace_back(tensor);
+static void LoadTensor(PATH_STRING_TYPE& pb_file, ONNX_NAMESPACE::TensorProto& input_pb) {
+  int tensor_fd;
```

Comment:
Can both these args be const? Same question for LoadSequenceTensor.

Reply:
Sure, but these diffs are misleading. This is the existing code as is. I can clean it up. (I didn't pay too much attention to already working code - I only spent time to adapt existing code )

Reply:
The first argument can be const, I would have thought the second one can't be given that is the instance whose fields we will be setting as part of the proto message parsing action...